### ISSUE TITLE
Add a new property "position" of notification

### ISSUE THREAD

**Issue by ankan17** [tjrexer/react-notifications#18]
Would love to be able to position the notification on different places on the screen e.g., TopLeft, TopRight, BottomLeft and BottomRIght.

**Comment by jalmeidanet** [comment]
Is this still undone?

**Comment by tjrexer** [comment]
Yes, @jalmeidanet this is still an open issue.  If you want to add the feature that would be great.

**Comment by jalmeidanet** [comment]
Ok, i will. Im having some problems building the project locally. Any thoughts about how to build and run it? Thanks.

**Comment by azubuike-nnamdi** [comment]
Hello @tjrexer , has this issue been resolved?
